nginx-shibboleth/tests/test_config.sls

```diff
@@ -0,0 +1,15 @@
+{% for fname in ['sp-key', 'sp-cert'] %}
+test_{{ fname }}_present:
+  testinfra.file:
+    - name: /etc/shibboleth/{{ fname }}.pem
+    - exists: True
+    - is_file: True
+    - owner:
+        expected: _shibd
+        comparison: eq
+    - group:
+        expected: _shibd
+        comparison: eq
+    - mode:
+        expected: 420
```

Comment:
Is this right as I don't believe it matches the permissions set [here](https://github.com/mitodl/nginx-shibboleth-formula/blob/blarghmatey/shibboleth_install/nginx-shibboleth/config.sls#L18)? Additionally, does [this](https://github.com/mitodl/nginx-shibboleth-formula/blob/blarghmatey/shibboleth_install/nginx-shibboleth/config.sls#L18) need to be 0644?